apps/cli/src/features/extract/entrypoint/extract/create-extract-draft-components-input.ts

```
@@ -1,4 +1,5 @@
 import type { ExtractDraftComponentsInput } from '@living-architecture/riviere-extract-ts-use-cases/features/extract/commands/extract-draft-components-input'
+import type { SourceFileSelection } from '@living-architecture/riviere-extract-ts-use-cases/features/extract/commands/extract-draft-components-input'
```

ESLint

[error] '`@living-architecture/riviere-extract-ts-use-cases/features/extract/commands/extract-draft-components-input`' imported multiple times.

(import/no-duplicates)